app/helpers/remote_execution_helper.rb

```diff
@@ -60,6 +60,12 @@ def job_invocation_task_buttons(task)
     job_invocation = task.task_groups.find { |group| group.class == JobInvocationTaskGroup }.job_invocation
     task_authorizer = Authorizer.new(User.current, :collection => [task])
     buttons = []
+    if (template = job_report_template) && authorized_for(controller: :report_templates, action: :generate)
+      buttons << link_to(_('Create Report'), generate_report_template_path(template, job_report_template_parameters(job_invocation, template)),
+        class: 'btn btn-default',
+        title: _('Create report for this job'),
+        disabled: task.pending?)
```

Comment:
If you go through monitor > report templates, then you can get a report generated for a still-running job, correct? It doesn't really care the job it is still running, it just generates the report from currently available data. Shouldn't we just allow doing the same through the button? I mean we should be consistent, not allow in through one place but not through the other

Reply:
I don't feel there is much value in doing so, is there ?
Would we want report on a still running task, do we have tasks that takes e.g. higher order of hours to complete, so someone would be reporting on that in the progress?
I believe we should be opinionated and allow user to do just what we believe has value. Otherwise we're confusing user with actions that doesn't make sense, but I'm not really able to say if it is valid usecase or not :thinking:

Reply:
Ok then, let's get it in and see what the users think